tools/clang/test/HLSLFileCheck/hlsl/workgraph/invalid_barrier_1.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: %dxv %s | FileCheck %s
+; RUN: %dxilver 1.8 | %dxv %s | FileCheck %s
```

Comment:
I'm surprised this is the only one that requires this. invalid_barrier_2/12.ll both use dx.op.barrierByMemoryType.

Reply:
Strange, you're right, how could these pass?  Maybe I missed something, let me double check.

Reply:
Yeah, there were more failures, working on it.  Basically, the failure I hit were the first ones and would crash the old validator, preventing more tests in the same group from running.  Then when testing locally, hcttest.cmd -dxil-loc mechanism has been completely broken by the lit testing addition.  So after fixing and trying this locally, everything passed, because it wasn't actually testing against the validator I selected.

Reply:
Ok, I've pushed a bunch of additional test fixes.  This should be good now.